dashboards-observability/public/components/trace_analytics/components/traces/__tests__/traces.test.tsx

```diff
@@ -33,6 +33,8 @@ describe('Traces component', () => {
         endTime="now"
         setEndTime={setEndTime}
         indicesExist={false}
+        hasTitle={true}
+        page="traces"
```

Comment:
you might want to add tests with both type of inputs for pages.

Reply:
Right, right. I'll add tests in a separate PR later.